test/e2e/storage/persistent_volumes-disruptive.go

```diff
@@ -230,10 +251,19 @@ func kubeletCommand(kOp kubeletOpt, c clientset.Interface, pod *v1.Pod) {
 	nodeIP, err := framework.GetHostExternalAddress(c, pod)
 	Expect(err).NotTo(HaveOccurred())
 	nodeIP = nodeIP + ":22"
-	sshResult, err := framework.SSH("sudo /etc/init.d/kubelet "+string(kOp), nodeIP, framework.TestContext.Provider)
-	Expect(err).NotTo(HaveOccurred())
+	sshResult, err := framework.SSH(fmt.Sprintf("sudo systemctl %s kubelet", string(kOp)), nodeIP, framework.TestContext.Provider)
+	Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Error occured attempting to SSH to node %q", pod.Spec.NodeName))
 	framework.LogSSHResult(sshResult)
-
+	if sshResult.Code != 0 {
+		framework.Logf("Restarting kubelet service via `systemctl` failed.  Attempting restart via `service`.")
```

Comment:
Service is correct here. `systemctl` was attempted and failed, so `service` is the fall back CLI attempt.